pkg/controller/operators/olm/apiservices.go

```diff
@@ -729,6 +731,19 @@ func (a *Operator) installAPIServiceRequirements(desc v1alpha1.APIServiceDescrip
 	// create a fresh CA bundle
 	apiService.Spec.CABundle = caPEM
 
+	// set apiservice owner to packageserver for proper cleanup on cluster upgrade
+	// only for packageserver api service
+	if apiService.Name == PackageserverName {
+		if l := apiService.GetLabels(); l != nil {
+			if apiService.Labels[ownerutil.OwnerKey] != ownerutil.OwnerPackageServer {
+				apiService.Labels[ownerutil.OwnerKey] = ownerutil.OwnerPackageServer
+			}
+		} else {
+			apiService.Labels = make(map[string]string)
+			apiService.Labels[ownerutil.OwnerKey] = ownerutil.OwnerPackageServer
+		}
```

Comment:
nit: doesn't `ownerutil.AddOwnerLabels(...)` handle this?

Reply:
Yes you're right, and we do so on L721. So I'm wondering if we need this code at all....